modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -66,6 +66,24 @@ The public key used in the instructions must correspond to the private key you c
 
 . Install the `docker` container runtime on your Windows VM following the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server[Microsoft documentation].
 
+. Install Microsoft container networking patch KB5012637 
+
+.. Download the patch files from the link:https://www.catalog.update.microsoft.com/Search.aspx?q=KB5012637[Microsoft Update Catalog].
```

Comment:
We can also give a powershell snippet to download the patch programmatically if you think that's beneficial:
```powershell
Invoke-WebRequest -Uri https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/updt/2022/04/windows10.0-kb5012637-x64_6a7459b60e226b0ad0d30b34a4be069bee4d2867.msu -OutFile windows10.0-kb5012637-x64.msu
```